moskito-backend/src/Entity/User.php

```diff
@@ -0,0 +1,164 @@
+<?php
+
+namespace App\Entity;
+
+use App\Repository\UserRepository;
+use Doctrine\Common\Collections\ArrayCollection;
+use Doctrine\Common\Collections\Collection;
+use Doctrine\ORM\Mapping as ORM;
+
+/**
+ * @ORM\Entity(repositoryClass=UserRepository::class)
+ */
+class User
+{
+    /**
+     * @ORM\Id
+     * @ORM\GeneratedValue
+     * @ORM\Column(type="integer")
+     */
+    private $id;
+
+    /**
+     * @ORM\Column(type="string", length=60)
+     */
+    private $firstName;
+
+    /**
+     * @ORM\Column(type="string", length=160)
+     */
+    private $lastName;
+
+    /**
+     * @ORM\Column(type="string", length=60)
+     */
+    private $email;
+
+    /**
+     * @ORM\Column(type="string", length=255)
+     */
+    private $password;
+
+    /**
+     * @ORM\ManyToMany(targetEntity=Campsite::class, inversedBy="users")
+     */
+    private $campsite;
+
+    /**
+     * @ORM\OneToMany(targetEntity=UserToken::class, mappedBy="user", orphanRemoval=true)
+     */
+    private $userTokens;
```

Comment:
I don't understand the plural here. Could a user have multiple (valid) tokens?

Reply:
Good point, at the moment, the user creates a new token everytime she logs in. Those are valid for one day. Maybe I find a way to delete old ones related to the user everytime a new one is created.